Description: In Debian, we don't/won't have lxc-android-config.
Author: Mike Gabriel <mike.gabriel@das-netzwerkteam.de>
Forwarded: not required

--- a/data/biometryd.service
+++ b/data/biometryd.service
@@ -1,7 +1,7 @@
 [Unit]
 Description=Mediate access to biometric devices
-Wants=lxc-android-config.service
-After=lxc-android-config.service
+#Wants=lxc-android-config.service
+#After=lxc-android-config.service
 
 [Service]
 Type=dbus
